apps/src/aichat/views/chatMessageShape.ts

```diff
@@ -0,0 +1,91 @@
+import PropTypes from 'prop-types';
+
+const chatMessageShape = PropTypes.shape({
```

Comment:
Is this used?

Reply:
@bencodeorg said it might be useful for him as he works on front-end client so i'll keep in place temporarily.